packages/devextreme/js/__internal/ui/m_validation_summary.ts

```diff
@@ -37,7 +37,7 @@ const ValidationSummary = CollectionWidget.inherit({
     const $element = this.$element();
     const group = this.option('validationGroup')
             || ValidationEngine.findGroup($element, this._modelByElement($element));
-    const groupConfig = ValidationEngine.addGroup(group);
+    const groupConfig = ValidationEngine.addGroup(group, false);
```

Comment:
[mostly memory leak]
I don't see there is a call of `removeGroup` in ValidationSummary => it's more safe to leave this group removable. Otherwise we just never remove this group..

Reply:
[ed5e8ca](https://github.com/DevExpress/DevExtreme/pull/27435/commits/ed5e8cae5a8f976fdac4650f50c05bc8bfe2b995)